core-services/image-mirroring/openshift/mapping_origin_4_5

```diff
@@ -145,6 +145,7 @@ registry.svc.ci.openshift.org/ocp/4.5:machine-config-operator-rpms quay.io/opens
 registry.svc.ci.openshift.org/ocp/4.5:machine-config-server quay.io/openshift/origin-machine-config-server:4.5 quay.io/openshift/origin-machine-config-server:4.5.0
 registry.svc.ci.openshift.org/ocp/4.5:managed-cluster-config quay.io/openshift/origin-managed-cluster-config:4.5 quay.io/openshift/origin-managed-cluster-config:4.5.0
 registry.svc.ci.openshift.org/ocp/4.5:managed-velero-operator quay.io/openshift/origin-managed-velero-operator:4.5 quay.io/openshift/origin-managed-velero-operator:4.5.0
+registry.svc.ci.openshift.org/ocp/4.5:csi-driver-manila quay.io/openshift/origin-csi-driver-manila:4.5 quay.io/openshift/origin-csi-driver-manila:4.5.0
```

Comment:
this is in the wrong place (needs to be alphabetical)

Reply:
my bad :( fixed